invenio_drafts_resources/services/draft.py

```diff
@@ -30,83 +27,55 @@ class DraftServiceConfig(RecordServiceConfig):
     resource_unit_cls = IdentifiedDraft
 
     # RecordService configuration
-    # TODO: FILL ME!
-
-    # DraftService configuration
-    # TODO: FILL ME!
-    draft_of_resource_unit_cls = IdentifiedRecord
-    draft_of_cls = Record
-    draft_cls = Draft  # As `service_object_cls`
     data_validator = MarshmallowDataValidator(
         schema=DraftMetadataSchemaJSONV1
     )
 
+    # DraftService configuration.
+    # WHY: We want to force user input choice here.
+    draft_cls = None
+
 
 class DraftService(RecordService):
     """Draft Service interface."""
 
     default_config = DraftServiceConfig
 
     # High-level API
-    def read(self, id_, identity):
-        """Retrieve a draft."""
-        # TODO: IMPLEMENT ME!
-        # High likelihood that we can just rely on RecordService here
-        # and simply configure DraftServiceConfig appropriately
-        return self.resource_unit_cls()
+    # Inherits record read, search, create, delete and update
 
-    def search(self, querystring, identity, pagination=None, *args, **kwargs):
-        """Search for drafts matching the querystring."""
-        # TODO: IMPLEMENT ME!
-        # High likelihood that we can just rely on RecordService here
-        # and simply configure DraftServiceConfig appropriately
-        return self.resource_list_cls()
+    def _index_draft(self, draft):
+        indexer = self.indexer()
+        if indexer:
+            indexer.index(draft)
 
-    def create(self, data, identity):
-        """Create a draft."""
-        # TODO: IMPLEMENT ME!
-        # # Check permissions: Can create draft of said record/resource
+    def create_new(self, data, identity):
```

Comment:
Discussed IRL:

- Naming of the service --> `RecordDraftService`
- `create_new` --> `create`
- `create_from` --> `edit` (creates a new draft)
- `edit` cannot be called `update` because the draft can still be *updated*